app/operators/mng-search.php

```diff
@@ -150,9 +151,9 @@
     }
 
     // setup php session variables for exporting
-    $_SESSION['reportTable'] = sprintf("%s AS rc LEFT JOIN %s AS ra ON ra.username=rc.username, %s AS ui",
+    $_SESSION['reportTable'] = sprintf("%s AS rc LEFT JOIN %s AS ra ON ra.username=rc.username, %s AS rr LEFT JOIN %s AS ui ON rr.username=ui.username",
                                        $configValues['CONFIG_DB_TBL_RADCHECK'], $configValues['CONFIG_DB_TBL_RADACCT'],
-                                       $configValues['CONFIG_DB_TBL_DALOUSERINFO']);
+                                       "radreply",$configValues['CONFIG_DB_TBL_DALOUSERINFO']);
```

Comment:
Hi @fercho1999leon , can you please amend the code here to not hardcode the "radreply" table and instead use the configValues definition for radreply?

Reply:
```suggestion
                                       $configValues['CONFIG_DB_TBL_RADREPLY'],$configValues['CONFIG_DB_TBL_DALOUSERINFO']);
```